### PR TITLE
Fix: import get_display

### PR DESCRIPTION
**Pull request by Yepness**
Fix get display import "from bidi import get_display" and no longer from "bidi.algorithm import get_display" this PR aim to fix that issue.

### CODE REVIEW

**Comment by Melanee-Melanee** [comment]
Please install this version of python-bidi:

!pip install python-bidi==0.4.2

**Comment by vinsunkavalli** [comment]
@Yepness I think the requirements also need to be changed to reflect that easyocr only supports newer versions of python-bidi if this change is implemented, no?

**Comment by Yepness** [comment]
@vinsunkavalli The current requirements reference the most current version of python-bidi, which includes the import adjustment. Therefore, there are two possible solutions:

1. Adjust the PR to be based on the most recent version of the repository.

2. Adjust the requirements to python-bidi==0.4.2 to use the import that is currently deprecated.

File: easyocr/easyocr.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
                    make_rotated_img_list, set_result_with_confidence,\
                    reformat_input_batched, merge_to_free
 from .config import *
-from bidi.algorithm import get_display
+from bidi import get_display
```
Comment (theta682) [inline easyocr/easyocr.py:9]:
```suggestion
try:
    from bidi import get_display
except:
    from bidi.algoritm import get_display
```

Comment (darrenchang) [reply easyocr/easyocr.py:9]: That seems like a good idea to ensure backward compability!

**Review by darrenchang — APPROVED** [review]
I like the fix

**Review by darrenchang — APPROVED** [review]
Can someone approve this

**Review by sagarishere — APPROVED** [review]
yup, do this.

This is a critical issue to make the base library to be useful out of the box.

**Comment by sagarishere** [comment]
@JaidedTeam

**Comment by rkcosmos** [comment]
Thanks everyone.

**Pull request merged**